src/settings/window_geometry.rs

```diff
@@ -13,6 +14,23 @@ pub const DEFAULT_WINDOW_GEOMETRY: Dimensions = Dimensions {
     height: 50,
 };
 
+#[derive(Serialize, Deserialize)]
+struct SerializedWindowSettings {
```

Comment:
I'm not sure about the "Serialized" prefix in name, here and in `SerializedSettings`. But I also don't have a better idea.